### DISCUSSION TITLE
Content Editor Layout Creation

### DISCUSSION THREAD

**Discussion by pnkr** [joomla/joomla-cms#31622] [Q&A] [answered]
It would be nice if joomla article editor had a more user friendly approach for content creation.
Now if someone wants to add a 2 or 3 column layout with image and content, that person needs to be very familiar with CSS and CSS Frameworks.

Developers and web designers are ok with this but for the end-user its a nightmare.

**Comment by bembelimen** [comment]
You can add predefined code snippets here: https://github.com/joomla/joomla-cms/tree/staging/media/editors/tinymce/templates and then your authors can select them with the tinyMCE via: 

![grafik](https://user-images.githubusercontent.com/1229869/101644549-9c9acb00-3a35-11eb-851b-dd98f21d2bf2.png)
and then:

![grafik](https://user-images.githubusercontent.com/1229869/101644603-af150480-3a35-11eb-9d92-7da2d9a09876.png)

**Comment by brianteeman** [comment, marked as answer]
This detailed blog post should help https://brian.teeman.net/joomla/883-multicolumn-editing-for-joomla-authors

**Comment by chmst** [comment]
@pnkr is this still an issue for you? Or can we close this?
